node_modules/oae-discussions/tests/test-discussions.js

```diff
@@ -2668,36 +2668,37 @@ describe('Discussions', function() {
 
             DiscussionsTestsUtil.setupMultiTenantPrivacyEntities(function(publicTenant, publicTenant1, privateTenant, privateTenant1) {
 
-                // Create message structure on the public discussion
-                RestAPI.Discussions.createMessage(publicTenant.loggedinUser.restContext, publicTenant.publicDiscussion.id, 'Message1 parent on public', null, function(err, publicMessage1) {
+                // Add a manager and member to the discussion
+                var updates = {};
```

Comment:
Would there be any value in running through the same test for a public, private and logged in discussion?